src/browser-frontend/actions/pages-model-actions.js

```diff
@@ -32,6 +32,8 @@ export default createActions({
     START_TAB_LOADED_ANIMATION: identity,
     STOP_TAB_LOADED_ANIMATION: identity,
     CLOSE_TAB_ANIMATED: identity,
+    MOVE_TAB_TO: identity,
+    SET_DRAGGING_TAB: identity,
```

Comment:
I'm strongly more in favor of having two actions for this: START_DRAGGING_TAB, STOP_DRAGGING_TAB, for a number of reasons. One would be so that action dispatchers don't have to know what a 'null/none dragging tab' is, and wouldn't need to know about the `NO_DRAGGING_TAB` constant when dispatching. Another reason would be for easier parsing of the redux action logs in the console by humans (this is why we don't use actions named like `TOGGLE`, or boolean `SET` etc.).